gossip/src/cluster_info.rs

```diff
@@ -4273,13 +4272,7 @@ mod tests {
         );
 
         let cluster_info44 = Arc::new({
-            let mut node = Node::new_localhost_with_pubkey(&keypair44.pubkey());
```

Comment:
this logic achieved nothing whatsoever, just rebinding gossip port for not reason

Reply:
I'm guessing this used to be some anti-gossip collision thing, but yeah... seems unnecessary now